Usually "linux" is default user in LiveCD/LiveDVD/LiveUSB of
openSUSE and openSUSE based appliances.

diff -ur a/src/modules/removeuser/removeuser.conf b/src/modules/removeuser/removeuser.conf
--- a/src/modules/removeuser/removeuser.conf
+++ b/src/modules/removeuser/removeuser.conf
@@ -10,4 +10,4 @@
 # continues as normal.
 ---
 # Username in the target system to be removed.
-username: live
+username: linux
